sdks/go/pkg/beam/options/jobopts/options.go

```diff
@@ -45,6 +46,12 @@ func init() {
 			"have no more than 1 override applied to it. If multiple "+
 			"overrides match a container image it is arbitrary which "+
 			"will be applied.")
+	flag.Var(&ResourceHints,
+		"resource_hints",
+		"Set whole pipeline level resource hints, accepting values of the format '<urn>=<value>'. "+
+			"Supports 'min_ram' and 'accelerator' hints. "+
```

Comment:
Optional opinionated feedback - IMO its not a great idea to specify the full set of acceptable values like that if the accepted values are liable to change without updating the code since its really easy for them to get out of date.

I'd personally prefer specifying these as example options

Reply:
Good call. Agreed. This was written before I learned that there's a "break glass" with default URN semantics.